src/main/java/CodeBuilder.java

```diff
@@ -962,7 +962,7 @@ public String credentialsTypeEquals(String given) {
     }
 
     public static String decodeJSON(String json) {
-        return json.replaceAll("&amp;quot;", "\"").replaceAll("&quot;", "\"");
+        return json.replaceAll("&amp;quot;", "\"").replaceAll("&quot;", "\"").replaceAll("&amp;", "&").replaceAll("&gt;", ">").replaceAll("<","&lt;").replaceAll("''", "'");
```

Comment:
I think moving the new substitutions into their own function would be better, since they are intended to decode the buildspecOverride YAML, not JSON.